index.js

```diff
@@ -18,27 +18,35 @@ var FILTER = filterize([
   /^\._/,
 ]);
 
-function pack(folder) {
-  var tarPack = tar.Pack({ noProprietary: true });
-  var contents = fstream.Reader({
-    path: folder,
-    type: 'Directory',
-    isDirectory: true,
-    filter: FILTER,
+function pack(folder, opts) {
+  opts = opts || {};
```

Comment:
If you use `util._extend({}, opts)` here, the line below won't modify the user's `opts` argument, which would be good.